Linux/sgx/libsgx_tsgxssl/Makefile

```
@@ -93,6 +93,10 @@ Sgx_tssl_Include_Paths := -I. -I$(PACKAGE_INC) -I$(SGX_SDK_INC) -I$(SGX_SDK_INC)
 Common_C_Cpp_Flags := -DOS_ID=$(OS_ID) $(SGX_COMMON_CFLAGS) -nostdinc -fdata-sections -ffunction-sections -Os -Wl,--gc-sections -fvisibility=hidden -fpie -fpic -fstack-protector -fno-builtin-printf -Wformat -Wformat-security $(Sgx_tssl_Include_Paths)
 Sgx_tssl_C_Flags := $(Common_C_Cpp_Flags) -Wno-implicit-function-declaration -std=c11 $(MITIGATION_CFLAGS) $(NO_THREADS_CFLAG)
 Sgx_tssl_Cpp_Flags := $(Common_C_Cpp_Flags) -std=c++11 -nostdinc++ $(MITIGATION_CFLAGS)
+ifeq ($(SKIP_INTELCPU_CHECK), TRUE)
```

Elsewhere, you seem to use an integer instead of `TRUE`:

https://github.com/intel/intel-sgx-ssl/blob/3c35551b8ab88da9b2f89b741969381628c76487/Linux/sgx/libsgx_tsgxssl/Makefile#L60-L61

```suggestion
ifeq ($(SKIP_INTELCPU_CHECK), 1)
```